### ISSUE TITLE
Can't find mackerel-plugin-sidekiq in released rpm package

### ISSUE THREAD

**Issue by mizukmb** [mackerelio/mackerel-agent-plugins#416]
I checked these rpm packages.

- [v0.33.0](https://github.com/mackerelio/mackerel-agent-plugins/releases/tag/v0.33.0)
  - mackerel-agent-plugins-0.33.0-1.el7.centos.x86_64.rpm
- [v0.28.1](https://github.com/mackerelio/mackerel-agent-plugins/releases/tag/v0.28.1)
  - mackerel-agent-plugins-0.28.1-1.el7.centos.x86_64.rpm
  - described in release note: `Add sidekiq plugin #354`

However, I couldn't find `mackerel-plugin-sidekiq`.

Environments:

```console
$ cat /etc/redhat-release
CentOS Linux release 7.3.1611 (Core)
$ rpm -q mackerel-agent-plugins
mackerel-agent-plugins-0.33.0-1.noarch
```

`sidekiq` couldn't be found at this file. Is this the issue?
https://github.com/mackerelio/mackerel-agent-plugins/blob/master/packaging/config.json

Thanks.

**Comment by syou6162** [comment]
@mizukmb I made PR to add mackerel-plugin-sidekiq to packages. 

- https://github.com/mackerelio/mackerel-agent-plugins/pull/417

We are planing to release next version's plugin package (including mackerel-plugin-sidekiq) within days 👍

**Comment by syou6162** [comment]
@mizukmb We have just released mackerel-agent-plugins v0.34.0 including mackerel-plugin-sidekiq!

**Comment by mizukmb** [comment]
@syou6162 I confirmed that mackerel-plugin-sidekiq is included at mackerel-agent-plugins v0.34.0 😄 
Thank you!
